Formula/libpeas.rb

```diff
@@ -10,18 +10,46 @@ class Libpeas < Formula
     sha256 "0f521913ca0eaf13b55aacb75e4b87a730be1f527215741ae2ba207caac523b2" => :el_capitan
   end
 
+  option "with-python", "install support for running Python plugins"
+
   depends_on "gettext" => :build
   depends_on "intltool" => :build
   depends_on "pkg-config" => :build
   depends_on "glib"
   depends_on "gobject-introspection"
   depends_on "gtk+3"
+  depends_on "python" => :optional
```

Comment:
Can this be built against the system `python`? If it can we could just turn it on by default, and leave the `python3` part as an option.

Reply:
My take would be to allow people to only install the languages they want/need. And the previous default was to build without any language support. In that light, I would suggest to keep the previous default behaviour.

Reply:
@c-alpha it doesn't add additional dependencies if we use system Python so we should just do that and make python3 optional.

```
depends_on "python" if MacOS.version <= :snow_leopard
```

Then always  build the python 2 bindings and build the python3 bindings if --with-python3 is passed too.

Reply:
Implemented with cb02cfdf69c2b4a501f3614463c3844ffff88925.